packages/esm-reports-app/src/components/edit-scheduled-report/edit-scheduled-report-form.scss

```
@@ -0,0 +1,60 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
```

```suggestion
@use '@carbon/layout';
@use '@carbon/type';
@use '@openmrs/esm-styleguide/src/vars' as *;
```

And then all the `spacing.$spacing-xx` tokens change to `layout.$spacing-xx`.

---

fixed